shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformViewsChannel.java

```diff
@@ -282,6 +296,15 @@ void resizePlatformView(
 
     /** Clears the focus from the platform view with a give id if it is currently focused. */
     void clearFocus(int viewId);
+
+    /**
+     * Whether the render surface of {@code FlutterView} should be converted to a {@code
+     * FlutterImageView} when a {@code PlatformView} is added.
+     *
+     * <p>This is done to syncronize the rendering of the PlatformView and the FlutterView.
+     * Defaults to true.
```

Comment:
nit: do you mean something like  "by default the Flutter frame is synchronized to the Android frame, but this method can be used to change the default behavior"?